src/main/java/com/prajina/academy/entity/Module.java

```diff
@@ -0,0 +1,93 @@
+package com.prajina.academy.entity;
+
+import java.util.Objects;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+
+@Entity
+@Table
+public class Module {
+	
+	@Id
+	// IDENTITY strategy is not supported in H2 db. Hence, using sequence. Auto was using the same sequence across all entities.
+	@GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "seq_module_id")
+	@SequenceGenerator(name = "seq_module_id", allocationSize = 1)
+	private Long id;
+	@Column(unique=true)
+	private String name;
```

Comment:
just a design thought here, you might want to have multiple modules of the same name, and use a version column to allow updating the module.